spec/factories/users.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user do
```

Comment:
FactoryBot allows you to specify aliases for a given factory. Therefore, if for example `user` is referred to as `creator` in your `post` model, then you'd need to indicate the alias here so FactoryBot knows that you'll be referring to `user` ans `creator` in the associated factories.

[`OPTIONAL`]

Reply:
Good to know, although at the moment I am not using a different name than a user. I will need it for the next model implementations